homeassistant/components/systemmonitor/sensor.py

```diff
@@ -333,21 +336,47 @@ async def async_setup_platform(
     config: ConfigType,
     async_add_entities: AddEntitiesCallback,
     discovery_info: DiscoveryInfoType | None = None,
+) -> None:
+    """Set up the system monitor sensors."""
+    async_create_issue(
+        hass,
+        DOMAIN,
+        "deprecated_yaml",
+        breaks_in_ha_version="2023.3.0",
```

Comment:
Minimum 2023.4.0 if this makes it in before beta on Wednesday. We want minimum two release cycles of deprecation time.